pkg/joinserver/grpc_deviceregistry.go

```diff
@@ -72,7 +74,10 @@ func (srv jsEndDeviceRegistryServer) Get(ctx context.Context, req *ttnpb.GetEndD
 		dev.RootKeys = &ttnpb.RootKeys{
 			RootKeyID: rootKeysEnc.GetRootKeyID(),
 		}
-		cc, _ := srv.JS.GetPeerConn(ctx, ttnpb.ClusterRole_CRYPTO_SERVER, dev.EndDeviceIdentifiers)
+		cc, err := srv.JS.GetPeerConn(ctx, ttnpb.ClusterRole_CRYPTO_SERVER, dev.EndDeviceIdentifiers)
+		if err != nil {
+			logger.WithError(err).Debug("Crypto Server connection is not available")
```

Comment:
OK so two things about this;

- Only log this if `!errors.IsNotFound(err)`
- I think it's a good practice to deliberately set `cc = nil` because imo the return values are undefined if `err != nil`